esdt.md

```diff
@@ -129,6 +129,46 @@ module ESDT
         <vmOutput> _ => VMOutput(OK, b"", .ListBytes, .List)</vmOutput>
 ```
 
+### Local Burn
+
+```k
+    syntax BuiltinFunction ::= "#ESDTLocalBurn"        [klabel(#ESDTLocalBurn), symbol]
+    rule toBuiltinFunction(F) => #ESDTLocalBurn requires F ==K #token("\"ESDTLocalBurn\"", "WasmStringToken")
+
+    rule [ESDTLocalBurn]:
+        <commands> processBuiltinFunction(#ESDTLocalBurn, SND, DST, <vmInput> 
+                                                                    <callValue> VALUE </callValue>
+                                                                    <callArgs> ARGS </callArgs>
+                                                                    _ 
+                                                                  </vmInput>)
+                => checkBool(VALUE ==Int 0, "built in function called with tx value is not allowed")
```

Comment:
How about "#ESDTLocalBurn" instead of "built in function" (the same for the log messages below)? When I'm debugging failures, the state I get is mostly empty (including the `<commands>` cell ), so it takes a while to figure out exactly what failed.

FWIW, the last similar failure I debugged was for a message that looked similar to "invalid arguments to process built-in function", which was caused by minting an amount that was too large, but it wasn't obvious what generated that error. It seems better to me to have more precise messages, e.g. "too many arguments to #ESDTLocalBurn" below and "argument too large for #ESDTLocalBurn" at line 150.

Reply:
Ah, ok, you're right.

Extra logging would be nice, if you have the time for it.